backend/src/test/scala/cromwell/backend/standard/callcaching/BlacklistCacheSpec.scala

```diff
@@ -1,22 +1,24 @@
 package cromwell.backend.standard.callcaching
 
 import cromwell.core.CacheConfig
+import cromwell.services.CallCaching.CallCachingEntryId
 import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
 
 import scala.concurrent.duration._
 import scala.language.postfixOps
 
 class BlacklistCacheSpec extends FlatSpec with BeforeAndAfterAll with Matchers {
+  val hit = CallCachingEntryId(3)
   "The blacklist cache" should "default, blacklist and expire" in {
     val bucket = "foo"
     val cacheConfig = CacheConfig(concurrency = 1, size = Integer.MAX_VALUE, ttl = 1 second)
     val cache = BlacklistCache(cacheConfig)
-    cache.isBlacklisted(bucket) shouldBe false
-    cache.blacklist(bucket)
-    cache.isBlacklisted(bucket) shouldBe true
+    cache.isBlacklisted(hit, bucket) shouldBe false
+    cache.blacklistBucket(bucket)
+    cache.isBlacklisted(hit, bucket) shouldBe true
 
     // Test ttl
     Thread.sleep(5000L)
```

Comment:
Might be nice to frame this as an `eventually` rather than a fixed 5-second pause.

Reply:
This is testing a specific value greater than the 1 second ttl specified above.